src/main/java/org/reaktivity/nukleus/sse/internal/stream/ServerStreamFactory.java

```diff
@@ -814,6 +836,73 @@ private void handleReset(
             final long traceId = reset.trace();
             doReset(applicationReplyThrottle, applicationRouteId, applicationReplyId, traceId);
         }
+
+        private void handleChallenge(
+            ChallengeFW challenge)
+        {
+            final HttpChallengeExFW httpChallengeEx = challenge.extension().get(httpChallengeExRO::tryWrap);
+            if (httpChallengeEx != null)
+            {
+                final JsonObject jsonHeaders = new JsonObject();
+                final JsonObject challengeJson = new JsonObject();
+                final ListFW<HttpHeaderFW> httpHeaders = httpChallengeEx.headers();
+
+                httpHeaders.forEach(header ->
+                {
+                    final String name = header.name().asString();
+                    final String value = header.value().asString();
+                    if (name != null)
+                    {
+                        if (!name.startsWith(":"))
```

Comment:
Keeping `name` as a `StringFW` means this check will need `name.sizeof() > 0 && name.buffer().get(name.offset()) == ASCII_COLON` where `ASCII_COLON` is a constant with value `0x3a`.